src/System.Text.Encodings.Web/src/Configurations.props

```diff
@@ -1,8 +1,9 @@
 ﻿<Project DefaultTargets="Build">
   <PropertyGroup>
     <BuildConfigurations>
-      netstandard1.0;
       netstandard;
+      netcoreapp;
+      uap-Windows_NT;
```

Comment:
Same here, do you actually need the netcoreapp and uap configurations? to me it looks in your .csproj that the output is pretty much the same so I would remove the configurations so that this is easier to service and that the package looks simpler.